Changes for aucore-blockvote-13: Must support & obligation

dt-r opened the pull request:
This PR addresses issues from the AU Core R1 Ballot for Comment aucore-blockvote-13: https://confluence.hl7.org/display/HAFWG/aucore-blockvote-13

Made the following changes :
- [FHIR-45231](https://jira.hl7.org/browse/FHIR-45231?src=confmacro)
- [FHIR-45195](https://jira.hl7.org/browse/FHIR-45195?src=confmacro)
- [FHIR-45163](https://jira.hl7.org/browse/FHIR-45163?src=confmacro)
- [FHIR-45103](https://jira.hl7.org/browse/FHIR-45103?src=confmacro)
- [FHIR-45095](https://jira.hl7.org/browse/FHIR-45095?src=confmacro)
- [FHIR-45073](https://jira.hl7.org/browse/FHIR-45073?src=confmacro)
- Updated change log

dbojicic reviewed (commented):
1. In 2.1.2.2 AU Core Capability Statements, change "a AU Core Requester actor" and 'a AU Core Responder actor" to use correct article "an". 

2. Home page - Capability Statements description should be updated to use Requester and Responder rather than Client and Server.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -315,28 +143,40 @@ The full set of sub-elements is visible in the "Key Elements Table" or "Snapshot
-Primitive elements are single elements with a primitive value. If they are marked as *Must Support*, then the AU Core Responder **SHALL** be capable of providing the element value to meet the *Must Support* requirement. AU Core Requestors **SHALL** be capable of processing a resource with all elements to meet the *Must Support* requirement.
+Primitive elements are single elements with a primitive value. If the primitive element is labelled as *Must Support*, then the 
+- AU Core Responder **SHALL** correctly populate the element if a value is known. 
+- AU Core Requesters **SHALL** accept resource without error if element present containing any valid value.
```
I find this hard to read - but that may be me only. Suggest chaning to something like:  
- AU Core Requesters **SHALL** accept _a_ resource without error if _the_ element is present _and contains_ ~~containing~~ any valid value.

dt-r replied:
is replaced with the exact text from the FHIR standard "resources containing any valid value for the element without error" also fixed equivalent sentence under the section "Must Support - Resource References"

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -315,28 +143,40 @@ The full set of sub-elements is visible in the "Key Elements Table" or "Snapshot
-- AU Core Requestors **SHALL** be capable of processing a Patient resource with a value in `Patient.name.family` and `Patient.name.given`.
+If any sub-element is labelled as *Must Support*, an: 
+- AU Core Responder **SHALL** correctly populate the element with all _Must Support_ sub-elements if a value is known. 
+- AU Core Requester **SHALL** accept resource without error if _Must Support_ sub_elements are present containing any valid value.
```
typo "sub_elements", change to "sub-elements"

dt-r replied:
fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -377,13 +224,16 @@ AU Core PractitionerRole|PractitionerRole.identifier|Medicare Provider Number
 {:.grid}
 
 For example, the profile [AU Core Patient](StructureDefinition-au-core-patient.html) constrains the choices for `Patient.identifier` defined in [AU Base Patient](https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-patient.html) to support Individual Healthcare Identifier (IHI), Medicare Card Number, Department of Veterans' Affairs (DVA) Number. When claiming conformance to the AU  ...
-- AU Core Responders **SHALL** be capable of populating `Patient.identifier` with an IHI, or Medicare Care Number, or DVA Number, or any combination of them
```
Full stop missing.

dt-r replied:
fixed

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -377,13 +224,16 @@ AU Core PractitionerRole|PractitionerRole.identifier|Medicare Provider Number
 
-A resource may support two elements that are used to indicate a reason, e.g. `Encounter.reasonCode` and `Encounter.reasonReference` in the profile [AU Core Encounter](StructureDefinition-au-core-encounter.html). In such cases, the server **SHALL** be able to populate at least one element, and the client application **SHALL** support processing resources with all supported elements. 
+A resource may support two elements that are used to indicate a reason, e.g. `Encounter.reasonCode` and `Encounter.reasonReference` in the profile [AU Core Encounter](StructureDefinition-au-core-encounter.html). In such cases, the
+- AU Core Responder **SHALL** correctly populate at least one element allowed by the profile, 
```
Replace comma at the end of the sentence with ".".

dt-r replied:
fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -396,9 +246,11 @@ AU Core Procedure|Procedure.reasonCode, Procedure.reasonReference
-In AU Core elements that define support for more than one terminology only apply to the [Coding](http://hl7.org/fhir/R4/datatypes.html#Coding) part of the element and are not intended to prevent systems from supplying only a text value. In such cases, the AU Core Responders **SHALL** be able to populate from at least one supported terminology, and AU Core Requesters **SHALL** support processing r ...
+In AU Core elements that define support for more than one code system only apply to the [Coding](http://hl7.org/fhir/R4/datatypes.html#Coding) part of the element and are not intended to prevent systems from supplying only a text value. In such cases, the 
+- AU Core Responders **SHALL** correctly populate the element from at least one supported code system
+- AU Core Requesters **SHALL** accept resources without error an element containing any valid code system. 
```
Incomplete sentence - suggest changing to something like: "AU Core Requesters SHALL accept resources without error if the element is present containing any valid code system."

dt-r replied:
replaced with "accept resources without error if the element is present and containing any valid value"

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
-- AU Core Requesters **SHALL** be capable of meaningfully processing a Medication resource with `Medication.code.coding` with both.
-- Systems **MAY** support populating and processing other terminologies but this is not a requirement of AU Core.
+- AU Core Responders **SHALL** correctly populate `Medication.code.coding` with either a code from [Australian Medication](https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1) or [PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-pbs-item.html), or both, if a coded value is known.
+- AU Core Requesters **SHALL** accept a Medication resource without error if `Medication.code.coding` present with either a code from [Australian Medication](https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1) or [PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-pbs-item.html), or both.
```
"if `Medication.code.coding` _+is+_ present ..."

dt-r replied:
fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+- AU Core Responder **SHALL NOT** populate the element in the resource when a value is not known.  
+
+When element definition is *Mandatory* (minimum cardinality > 0), 
+- AU Core Responder **SHALL** correctly populate the element *even when* the system does not know a value or the reason a the value is absence. 
```
Fix "a the". Also need to make some minor grammar adjustments; suggest something like: 
- AU Core Responder **SHALL** correctly populate the element *even when* the system does not know a value or the reason for its absence.

dt-r replied:
This text is no longer present.  but re-wrote slightly what is in master to improve it in this branch.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+
+1.  For *non-coded* data elements including type [Reference](http://hl7.org/fhir/R4/references.html#Reference), 
+  - when the AU Core profile for the element does not require a sub-element
+    - use the [DataAbsentReason extension](http://hl7.org/fhir/R4/extension-data-absent-reason.html) in the data type 
```
increase indentation in this and the bullet point below.

dt-r replied:
The exact layout is nolonger present. Re-review the changed text to see what issues are extant.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+          }
+      ~~~
+      
+  - when the AU Core profile mandates a sub-element, such as a valid Identifier or Reference data type, then the resource must contain the sub-element otherwise the resource will not be conformant,
```
Is this meant be a bullet point in _1. For non-coded data elements..._? Or a separate requirement? It is sort of unclear where it belongs. It also seems to reiterate information that's in "Must Support - Complex Elements" without any specific Missing/Suppressed Data requirements which does is not an issue but suggest changes to improve the sentence: 
- change comma at the end of the sentence to full stop
- clarify that sub-elements can be of data types Identifier or Reference rather than being the Identifier or Reference data type itself

dt-r replied:
The exact layout is nolonger present. Re-review the changed text to see what issues are extant.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+1. For *coded* data elements:
+    - *required* binding strength (CodeableConcept or code datatypes):
+      - the appropriate "unknown" concept code **SHALL** be populated if available.
+      - For AU Core profiles, the following *mandatory* or *conditionally mandatory* status elements with required binding have no appropriate "unknown" concept code:
```
"mandatory" and "conditionally mandatory" are italicised and am sure there will be questions around this especially conditionally mandatory. Suggest adding the additional content from US Core to improve clarity:
- add * AllergyIntolerance.clinicalStatus, and Condition.clinicalStatus in the bullet point list, and
- add "*The clinicalStatus element is conditionally mandatory based on resource-specific constraints."

dt-r replied:
restored the text "*The clinicalStatus element is conditionally mandatory based on resource-specific constraints." that was in the ballot for comment version of AU Core.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+  - when the AU Core profile mandates a sub-element, such as a valid Identifier or Reference data type, then the resource must contain the sub-element otherwise the resource will not be conformant,
+
+1. For *coded* data elements:
+    - *required* binding strength (CodeableConcept or code datatypes):
```
Change to "CodeableConcept, or code datatypes" to make it consistent with other bullet point (example, preferred...bindings).

dt-r replied:
No change. This is not desired. Could list all three code types but prefer the short hand of 'coded'.

dbojicic replied:
My comment wasn't clear enough. This was simply about adding a comma so that "(CodeableConcept or code datatypes)" changes to "(CodeableConcept, or code datatypes)". 
To make it consistent with "(CodeableConcept, or Coding datatypes)".

dt-r replied:
removed both (CodeableConcept, or Coding datatypes) and (CodeableConcept or code datatypes) - they weren't necessary.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+      - when the system has text but no coded value, only the text sub-element is populated.
+      - when there is neither text or coded value:
+        - the appropriate "unknown" concept code **SHALL** be populated if available when the binding strength is *extensible*
+        - when the value set does not have an appropriate "unknown" concept code, use `unknown` from the [DataAbsentReason Code System](http://terminology.hl7.org/CodeSystem/data-absent-reason).
```
Missing conformance verb.

dt-r replied:
removed the one conformance verb so it doesn't confuse a reader of this section.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+        - `Condition.clinicalStatus`
+        - `Immunization.status`
+    - *example*, *preferred*, or *extensible* binding strengths (CodeableConcept, or Coding datatypes):
+      - when the system has text but no coded value, only the text sub-element is populated.
```
missing conformance verb

dt-r replied:
removed conformance verbs from this section instead

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+    - *example*, *preferred*, or *extensible* binding strengths (CodeableConcept, or Coding datatypes):
+      - when the system has text but no coded value, only the text sub-element is populated.
+      - when there is neither text or coded value:
+        - the appropriate "unknown" concept code **SHALL** be populated if available when the binding strength is *extensible*
```
Missing full stop.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+                  {
+                    "system" : "http://terminology.hl7.org/CodeSystem/data-absent-reason",
+                    "code" : "unknown",
+                    "display" : "unknown"
```
Correct display to "Uknown".

dt-r replied:
fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
+The [AU Core Requester CapabilityStatement](CapabilityStatement-au-core-requester.html) defines the conformance requirements and expectations of a [AU Core Requester](ActorDefinition-au-core-actor-requester.html) actor responsible for initiating queries for information from an [AU Core Responder](ActorDefinition-au-core-actor-responder.html). The complete list of FHIR profiles, REST API interacti ...
 
-The [AU Core Server CapabilityStatement](CapabilityStatement-au-core-server.html) outlines conformance requirements and expectations for an AU Core Server. This CapabilityStatement describes the basic rules for the AU Core Responder actor that is responsible for providing responses to queries submitted by AU Cores Requestors. The complete list of FHIR profiles, RESTful operations, and search para ...
+The [AU Core Responder CapabilityStatement](CapabilityStatement-au-core-responder.html) defines the conformance requirements and expectations of a [AU Core Responder](ActorDefinition-au-core-actor-responder.html) actor responsible for providing responses to queries submitted by an [AU Core Requester](ActorDefinition-au-core-actor-requester.html). The complete list of FHIR profiles, REST API inter ...
```
typo "capbilities"

dt-r replied:
fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
+A system that [declares conformance](declaring-conformance.html#profile-only-support) as _Profile Only Support_ to an AU Core profile:
+- **SHALL** correctly populate a resource with all mandatory elements specified in the profile’s definition.
+- **SHALL** correctly populate a resource with all _Must Support_ elements if a value is known for the element. 
+- **SHALL** implement the requirements on [Missing data](general-requirements.html#missing-data) when an element value is not known.
```
Change link text "Missing data" to "Missing Data" to make it consistent with the section title.

dt-r replied:
fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
-- **SHALL** populate all elements labelled *Must Support* unless:
-   - the element is not available for that patient.
-   - a requester does not have access rights to the element (the data is suppressed due to a security or privacy reason), in this case the responder **SHALL** implement the requirements on [Suppressed Data](general-requirements.html#suppressed-data).
-- **SHALL** implement the requirements on [Missing data](general-requirements.html#missing-data) when an element is not available for a patient.
```
Change link text "Missing data" to "Missing Data" to make it consistent with the section title.

dt-r replied:
fixed in 4 instances.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
-   - a requester does not have access rights to the element (the data is suppressed due to a security or privacy reason), in this case the responder **SHALL** implement the requirements on [Suppressed Data](general-requirements.html#suppressed-data).
-- **SHALL** implement the requirements on [Missing data](general-requirements.html#missing-data) when an element is not available for a patient.
-  - **SHALL NOT** include an element in the resource instance returned as part of query results when the element is optional.
-  - **SHALL** include an element in the resource instance returned as part of query results when the element is mandatory by following the instructions in the section on [Missing data](general-requirements.html#missing-data).
```
Change link text "Missing data" to "Missing Data" to make it consistent with the section title.

dt-r replied:
fixed

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
-- **MAY** choose to reject non-conformant resources 
-- **SHALL** interpret missing data elements within resource instances as data not present in the source system when querying AU Core Responders
+- **SHALL** accept resources containing any valid value for _Must Support_ elements without error.
+- **SHALL** accept resources containing _Must Support_ elements with [Missing data](#missing-data) or [Suppressed data](#suppressed-data) without error.
```
Change link text "Missing data" to "Missing Data" to make it consistent with the section title.

dt-r replied:
fixed

dbojicic commented on input/resources/au-core-actor-responder.xml:
```diff
@@ -0,0 +1,16 @@
+  <documentation value="An AU Core Responder, also known as AU Core Server:&#xa;&#xa;
+- **SHALL** all *Must Support* elements if they know a value for the element&#xa;&#xa;
+- **SHALL** implement the requirements on [Suppressed Data](general-requirements.html#suppressed-data) when an element is NOT allowed to be shared&#xa;&#xa;
+- **SHALL** implement the requirements on [Missing data](general-requirements.html#missing-data) when an element value is NOT known:&#xa;&#xa;
```
Change link text "Missing data" to "Missing Data" to make it consistent with the section title.

dt-r replied:
fixed

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -315,28 +143,40 @@ The full set of sub-elements is visible in the "Key Elements Table" or "Snapshot
 
-- AU Core Responders **SHALL** be capable of providing a value in `Organization.name`.
-- AU Core Requestors **SHALL** be capable of processing an Organization resource with a value in `Organization.name`.
+- AU Core Responder **SHALL** correctly populate a value in `Organization.name` if a value is know.
```
typo know" - change to "known"

dt-r replied:
fixed

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -377,13 +224,16 @@ AU Core PractitionerRole|PractitionerRole.identifier|Medicare Provider Number
-- AU Core Requesters **SHALL** be capable of meaningfully processing a Patient resource populated with an IHI, Medicare Care Number, and DVA Number.
-- Systems **MAY** support populating and processing other identifiers, but this is not a requirement of AU Core.
+- AU Core Responder **SHALL** correctly populate `Patient.identifier` with an IHI, or Medicare Care Number, or DVA Number, or any combination of them
+- AU Core Requester **SHALL** accept Patient resource resource if `Patient.identifier` is present containing any IHI, Medicare Care Number, or DVA Number identifier type.
```
duplicated "resource" - remove one instance

dt-r replied:
fixed

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -1,8 +1,8 @@
+1. [Profile Only Support](#profile-only-support): system that represent digital health information using AU Core profiles without the expectation to implement any FHIR specified interactions.
+1. [Profile Support + Interaction Support](#profile--interaction-support): system that represents digital health information using AU Core profiles (Profile Support) AND implement the [FHIR REST API](http://hl7.org/fhir/R4/http.html) interactions defined by an [AU Core Capability Statement](#au-core-capability-statements) (Interaction Support).
 
 The requirements of the FHIR standard and [FHIR Conformance Rules](http://hl7.org/fhir/conformance-rules.html) apply, and define the use of terms in this guide including the conformance verbs - SHALL, SHALL NOT, SHOULD, SHOULD NOT, MAY.
```
Use the conformance verb formatting for conformance verbs for the listed conformance.

dt-r replied:
Changed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
+_Profile + Interaction Support_ refers to a system that represents digital health information using the content models of AU Core profiles AND implement the REST API interactions defined by the applicable AU Core capability statement.
 
-For elements with a minimum cardinality = 0 (including elements labelled *Must Support*), the element **SHALL** be omitted from the resource if they are suppressed.
+An [AU Core Responder](ActorDefinition-au-core-actor-responder.html) that declares 'Profile + Interaction Support' conformance:
```
Different formatting here: 'Profile + Interaction Support' as opposed to using italics in 2.1.2.3 Profile Only Support:
"A system that [declares conformance](https://github.com/hl7au/au-fhir-core/pull/168/declaring-conformance.html#profile-only-support) as _Profile Only Support_ to an AU Core profile:"

Apply consistent formatting. 

There is one more instance of 'Profile + Interaction Support' below.

dt-r replied:
All instances of "Profile + Interaction Support" fixed to be "Profile Support + Interaction Support".
Removed formatting of either italics or single comma.

Also fixed the wording that had referred to these as 'conformance tiers'. They are not; no such concept is defined.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
+Labelling an element *[Must Support](https://www.hl7.org/fhir/conformance-rules.html#mustSupport)* means that systems that produce or consume resources **SHALL** provide support for the element in some meaningful way. The FHIR standard does not define exactly what 'meaningful' support for an element means, but indicates that a profile **SHALL** make clear exactly what kind of support is required  ...
 
-Because AU Core is a foundational standard, *Must Support* needs to be defined a way that does not impede or prescribe what a client or server does with the data, so as not to impede each implementation’s ability to tighten and define expectations for use under their own business rules, regulations, policies, etc. There is also a the challenge that comes from inheritance of must support flags int ...
+Because AU Core is a foundational standard, *Must Support* needs to be defined a way that does not impede or prescribe what a system does with the data, so as not to impede each implementation’s ability to tighten and define expectations for use under their own business rules, regulations, policies, etc. There is also a the challenge that comes from inheritance of must support flags into implemen ...
```
1. typo 'a the' 
2. I think this can also be broken into two sentences: 
There is also a challenge that comes from inheritance of must support flags into implementation guides that have strict definitions for must support (e.g., must be able to display this value to an end user). AU Core will only apply the *Must Support* flag on the elements that are necessary to support _minimum_ requirements and are expected to be flagged as *Must Support* across the majority of Australian FHIR implementation guides.

dt-r replied:
1. fixed
2. fixed

dbojicic reviewed (commented):
Home page - How to read this guide should be updated to include two new pages: Declaring Conformance and Actor Definitions.

dbojicic reviewed (commented):
Actor Definitions menu item displays a full Artifacts Summary page. Unsure if this is intentional but find it unexpected and potentially confusing - suggest updating a page to include only the actual ActorDefinitions and related information only.

dbojicic commented on input/resources/au-core-actor-requester.xml:
```diff
@@ -0,0 +1,16 @@
+  <name value="AUCoreRequester"/> 
+  <title value="AU Core Requester"/> 
+  <status value="draft"/>
+  <description value="The AU Core Requester is a system that responds to the data access request submitted by requesters, providing responses to requests for core digital health and administrative information."/> 
```
"The AU Core Requester is a system that **responds**...." - correct to state this is a system that creates and initiates a data access request.

dt-r replied:
fixed.

dbojicic commented on input/resources/au-core-actor-responder.xml:
```diff
@@ -0,0 +1,16 @@
+  <description value="The AU Core Responder is a system responsible for providing responses to queries submitted by AU Core Requesters."/> 
+  <type value="system"/> 
+  <documentation value="An AU Core Responder, also known as AU Core Server:&#xa;&#xa;
+- **SHALL** all *Must Support* elements if they know a value for the element&#xa;&#xa;
```
Missing verb - "correctly populate".

dt-r replied:
fixed

dbojicic commented on input/resources/au-core-actor-responder.xml:
```diff
@@ -0,0 +1,16 @@
+  <documentation value="An AU Core Responder, also known as AU Core Server:&#xa;&#xa;
+- **SHALL** all *Must Support* elements if they know a value for the element&#xa;&#xa;
+- **SHALL** implement the requirements on [Suppressed Data](general-requirements.html#suppressed-data) when an element is NOT allowed to be shared&#xa;&#xa;
+- **SHALL** implement the requirements on [Missing data](general-requirements.html#missing-data) when an element value is NOT known:&#xa;&#xa;
```
Remove ":" from the end of the sentence

dt-r replied:
fixed

dbojicic commented on input/resources/au-core-actor-requester.xml:
```diff
@@ -0,0 +1,16 @@
+  <documentation value="An AU Core Client, also known as AU Core Requester:&#xa;&#xa;
+- **SHALL** accept resources containing any valid value for *Must Support* elements without error&#xa;&#xa;
+- **SHALL** accept resources containing *Must Support* elements with [Missing data](general-requirements.html#missing-data) or [Suppressed Data](general-requirements.html#suppressed-data) without error&#xa;&#xa;
+How the system processes the resource depends on local requirements that could align with obligation terms such [reject invalid](https://hl7.org/fhir/extensions/CodeSystem-obligation.html#obligation-reject-invalid), [correctly handle](https://hl7.org/fhir/extensions/CodeSystem-obligation.html#obligation-handle), [persist](https://hl7.org/fhir/extensions/CodeSystem-obligation.html#obligation-persi ...
```
A word is missing; change to: "How the system processes the resource depends on local requirements that could align with obligation terms such _+as+_ [reject invalid](https://hl7.org/fhir/extensions/CodeSystem-obligation.html#obligation-reject-invalid), ..."

dt-r replied:
fixed

dbojicic reviewed (commented):
Obligations on the Must Support elements inherited from core FHIR (applies to all vital signs profiles) are missing. Please add.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
+[AU Core Responder](ActorDefinition-au-core-actor-responder.html) | [SHALL:populate-if-known](https://hl7.org/fhir/extensions/CodeSystem-obligation.html#obligation-SHALL.58populate-if-known) | SHALL populate if known | Conformant applications producing resources SHALL correctly populate this element if they know a value for the element, but it is acceptable if the system is unable to ever know a  ...
+[AU Core Requester](ActorDefinition-au-core-actor-requester.html) | [SHALL:no-error](https://hl7.org/fhir/extensions/CodeSystem-obligation.html#obligation-SHALL.58no-error) | SHALL not error if present | Conformant applications SHALL accept resources containing any valid value for the element without error. | Applications are still able to inform the user that a value cannot be processed correctl ...
+
+Must Support elements are treated differently between [AU Core Responder](ActorDefinition-au-core-actor-responder.html) and [AU Core Requester](ActorDefinition-au-core-actor-requester.html) actors, *Must Support* on a profile element **SHALL** be interpreted as follows.
```
This sentence can be made clearer if broken into two sentences: "Must Support elements are treated differently between [AU Core Responder](ActorDefinition-au-core-actor-responder.html) and [AU Core Requester](ActorDefinition-au-core-actor-requester.html) actors. *Must Support* on a profile element **SHALL** be interpreted as follows."

dt-r replied:
changed.

dbojicic commented on input/pagecontent/general-requirements.md:
"Profiles defined in this implementation publication flag *Must Support* on elements **and not part sub-elements of a data type.**" - not true anymore

dt-r replied:
fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
 #### AU Core Profiles and Extensions
 The [Profiles and Extensions](profiles-and-extensions.html) page lists the AU Core profiles and AU Core extensions defined for this implementation guide. An AU Core profile [StructureDefinitions](http://hl7.org/fhir/R4/structuredefinition.html) defines the minimum elements, extensions, vocabularies and value sets that **SHALL** be present and constrains the way elements are used when conforming t ...
```
fix grammar - 'defines' should be 'define'

dt-r replied:
No, it is correct in that 'defines' is current tense rather than 'will define' or future tense 'defined'.

dbojicic replied:
Yes. I know why I misread it - [StructureDefinitions](http://hl7.org/fhir/R4/structuredefinition.html) hyperlink text should be StructureDefinition.

dt-r replied:
This is actually handled by a separate Jira - so we'll leave it that PR.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
+- **SHALL** implement the required REST API interactions and search parameters as defined in the [AU Core Requester Capability Statement](CapabilityStatement-au-core-requester.html#resourcesSummary1).
 
 ### Mandatory Elements
 Mandatory elements are elements with a minimum cardinality of 1 (min=1). When an element is Mandatory, the data is expected to always be present. Very rarely, it may not be, and in this circumstance the requirements in [Missing Data](#missing-data) **SHALL** be applied. An element can be both _Must Support_ and mandatory, in which case the requirements for mandatory's Missing data requirements ap ...
```
Missing Must Support and Mandatory Data section does not exist anymore.

dt-r replied:
That is a bad merge change. That section is in master and should be present in this branch.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+When an element definition is *optional* (minimum cardinality = 0), including elements labelled *Must Support*, an
+- AU Core Responder **SHALL NOT** populate the element in the resource when a value is suppressed.  
+
+When an element definition is *Mandatory* (minimum cardinality > 0), 
```
Optional and mandatory have already been defined in the content above, which I don't think is an issue. Maybe consider defining Mandatory in the same way (defined as both minimum cardinality =1 and minimum cardinality >0) for consistency. 
Same applies to Missing Data section as well.

dt-r replied:
No change. The ballot approved text has been restored. This will be retained.

dbojicic commented:
Overall formatting of Mandatory & optional on the page is inconsistent: _Mandatory_, Mandatory, _mandatory_, mandatory, _optional_. I've tried to understand the pattern (beyond just wanting to highlight optional vs mandatory) but could not. The presence of formatting variations suggests that it's important, but the inconsistency makes it unclear. Suggest italicising all instances?

dbojicic commented on input/resources/au-core-requester.xml:
```diff
@@ -1,10 +1,10 @@
-      <h2 id="title">AU Core Client CapabilityStatement</h2>
+      <h2 id="title">AU Core Requester CapabilityStatement</h2>
       <ul>
         <li>Implementation Guide Version: 0.3.0-ci-build</li>
```
Incorrect version number; should be 0.4.0-ci-build or left as TBD.

dt-r replied:
Not in scope of this PR.

dbojicic commented on input/resources/au-core-responder.xml:
```diff
@@ -1,10 +1,10 @@
-      <h2 id="title">AU Core Server CapabilityStatement</h2>
+      <h2 id="title">AU Core Responder CapabilityStatement</h2>
       <ul>
         <li>Implementation Guide Version: 0.3.0-ci-build</li>
```
Incorrect version number; should be 0.4.0-ci-build or left as TBD.

dt-r replied:
Not in scope of this PR.

dbojicic reviewed (changes requested):
Please see comments.

dt-r commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
-  - **SHALL** include an element in the resource instance returned as part of query results when the element is mandatory by following the instructions in the section on [Missing data](general-requirements.html#missing-data).
+- **SHALL** correctly populate all _Must Support_ elements if a value is known for the element.
+- **SHALL** implement the requirements on [Suppressed Data](general-requirements.html#suppressed-data) when an element is NOT allowed to be shared.
+- **SHALL** implement the requirements on [Missing data](general-requirements.html#missing-data) when an element value is not known.
```
fixed

dt-r commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
-For mandatory elements (minimum cardinality is > 0), the element **SHALL** be populated but it may exceed the data requester's access rights to know that the data is suppressed:
-- where a requester does not have access rights to know that data is suppressed use the code `unknown` from the [DataAbsentReason Code System](http://terminology.hl7.org/CodeSystem/data-absent-reason) following the section on [Missing Data](#missing-data).
-- where a requester may know that the data is suppressed use the code `masked` from the [DataAbsentReason Code System](http://terminology.hl7.org/CodeSystem/data-absent-reason) following the section on [Missing Data](#missing-data).
+An [AU Core Requester](ActorDefinition-au-core-actor-requester.html) that declares 'Profile + Interaction Support' conformance:
```
See comment above on fix.

dt-r commented:
> Actor Definitions menu item displays a full Artifacts Summary page. Unsure if this is intentional but find it unexpected and potentially confusing - suggest updating a page to include only the actual ActorDefinitions and related information only.

It is intentional. I will take this as a review comment to make a change.

dt-r commented on input/resources/au-core-actor-requester.xml:
```diff
@@ -0,0 +1,16 @@
+  <type value="system"/> 
+  <documentation value="An AU Core Client, also known as AU Core Requester:&#xa;&#xa;
+- **SHALL** accept resources containing any valid value for *Must Support* elements without error&#xa;&#xa;
+- **SHALL** accept resources containing *Must Support* elements with [Missing data](general-requirements.html#missing-data) or [Suppressed Data](general-requirements.html#suppressed-data) without error&#xa;&#xa;
```
fixed

dt-r commented:
> Overall formatting of Mandatory & optional on the page is inconsistent: _Mandatory_, Mandatory, _mandatory_, mandatory, _optional_. I've tried to understand the pattern (beyond just wanting to highlight optional vs mandatory) but could not. The presence of formatting variations suggests that it's important, but the inconsistency makes it unclear. Suggest italicising all instances?

Raise a change request or technical correction.

dbojicic commented:
> > Overall formatting of Mandatory & optional on the page is inconsistent: _Mandatory_, Mandatory, _mandatory_, mandatory, _optional_. I've tried to understand the pattern (beyond just wanting to highlight optional vs mandatory) but could not. The presence of formatting variations suggests that it's important, but the inconsistency makes it unclear. Suggest italicising all instances?
> 
> Raise a change request or technical correction.

Raised https://jira.hl7.org/browse/FHIR-45993.

dt-r commented:
> Home page - How to read this guide should be updated to include two new pages: Declaring Conformance and Actor Definitions.

fixed.

dt-r commented:
> 1. In 2.1.2.2 AU Core Capability Statements, change "a AU Core Requester actor" and 'a AU Core Responder actor" to use correct article "an".
> 2. Home page - Capability Statements description should be updated to use Requester and Responder rather than Client and Server.

1. Fixed.
2. Fixed

dt-r commented on input/pagecontent/general-requirements.md:
```diff
@@ -396,9 +246,11 @@ AU Core Procedure|Procedure.reasonCode, Procedure.reasonReference
 
-In AU Core elements that define support for more than one terminology only apply to the [Coding](http://hl7.org/fhir/R4/datatypes.html#Coding) part of the element and are not intended to prevent systems from supplying only a text value. In such cases, the AU Core Responders **SHALL** be able to populate from at least one supported terminology, and AU Core Requesters **SHALL** support processing r ...
+In AU Core elements that define support for more than one code system only apply to the [Coding](http://hl7.org/fhir/R4/datatypes.html#Coding) part of the element and are not intended to prevent systems from supplying only a text value. In such cases, the 
+- AU Core Responders **SHALL** correctly populate the element from at least one supported code system
```
fixed.

dt-r commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+      
+  - when the AU Core profile mandates a sub-element, such as a valid Identifier or Reference data type, then the resource must contain the sub-element otherwise the resource will not be conformant,
+
+1. For *coded* data elements:
```
fixed

dt-r commented on input/pagecontent/general-requirements.md:
```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+When an element definition is *optional* (minimum cardinality = 0), including elements labelled *Must Support*, an
+- AU Core Responder **SHALL NOT** populate the element in the resource when a value is suppressed.  
+
+When an element definition is *Mandatory* (minimum cardinality > 0), 
```
This text is nolonger present - replaced with the change from master.

dt-r commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
 #### Profile Only Support
-Systems may deploy, and support, one or more AU Core profiles to represent digital health information. A system that implements _profile only support_ will use a profile's content model without any expectations to implement AU Core interactions and:
-- **SHALL** populate a resource with all profile elements that are mandatory by that profile’s StructureDefinition.
-- **SHALL** populate a resource with all profile elements labelled Must Support unless that element is not available. Where the data is not available the system **SHALL** implement the requirements on [Missing data](general-requirements.html#missing-data).
```
fixed.

dt-r commented:
> Obligations on the Must Support elements inherited from core FHIR (applies to all vital signs profiles) are missing. Please add.

Added, noting that this was not added to Observation.value[x] as it is irrelevant in the profiles. It was added to Observation.valueQuantity or in the case of Blood Pressure it was not added since that profile prohibits the element Observation.value

dbojicic commented on input/pagecontent/index.md:
```diff
@@ -72,21 +72,21 @@ implementation. This implementation guide will be used as the basis for Australi
 
 ### AU Core FHIR RESTful interactions
 
-For systems that support AU Core FHIR RESTful interactions ([Profile + Interaction Support](general-requirements.html#profile--interaction-support)), the interaction requirements, such as read and search operations, are detailed in the AU Core Requester and AU Core Responder [CapabilityStatements](capability-statements.html).
```
"Profile + Interaction Support" hyperlink text should be updated to "Profile Support + Interaction Support" in line with the changes in general-requirements.

dt-r replied:
fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,76 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
-* The server hosts a [capability statement](http://hl7.org/fhir/capabilitystatement.html) at [url]/metadata that is available to both authenticated and unauthenticated clients that declares the profiles supported as either a base profile using [CapabilityStatement.rest.resource.profile](http://hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.rest.resource.profile) or supporte ...
+An [AU Core Responder](ActorDefinition-au-core-actor-responder.html) that declares Profile Support + Interaction Support conformance:
+- **SHALL** [declare conformance](declaring-conformance.html#profile-support--interaction-support) to the [AU Core Responder Capability Statement](CapabilityStatement-au-core-responder.html) in the server's CapabilityStatement.
+- **SHALL** specify the implemented resource types, associated AU Core profiles, REST API interactions and search parameters in the server's CapabilityStatement
```
Missing full stop at the end of a few bullet points here

dt-r replied:
fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -18,246 +18,76 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
 
-    Example: CapabilityStatement resource for a server supporting the AU Core Patient profile as a system-wide profile that is applied across all instances of the Patient resource:
+An [AU Core Requester](ActorDefinition-au-core-actor-requester.html) that declares Profile Support + Interaction Support conformance:
+- **SHALL** implement the [Mandatory](#mandatory-elements) and/or [Must Support](#must-support-and-obligation) requirements for the elements defined as such in the implemented AU Core profiles
```
Missing full stop.

dt-r replied:
fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -300,7 +130,7 @@ Example: AU Core AllergyIntolerance profile showing clinicalStatus and verificat
 #### Interpreting profile elements labelled Must Support
 
-Profiles defined in this implementation publication flag *Must Support* on elements and not part sub-elements of a data type. 
+Profiles defined in this implementation publication flag *Must Support* on elements (e.g. `Patient.name`) and part sub-elements of a data type (e.g. `Patient.name.use`). 
```
I don't think the term "part sub-elements" is needed here and can be confusing. Can simply say "sub-elements of a data type (e.g. `Patient.name.use`)"

dt-r replied:
fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -315,28 +145,39 @@ The full set of sub-elements is visible in the "Key Elements Table" or "Snapshot
+- AU Core Responders **SHALL** correctly populate the element with all _Must Support_ sub-elements for which a value is known. 
+- AU Core Requesters **SHALL** accept resource without error if _Must Support_ sub-elements are present and containing any valid value.
+
+For example, in the AU Core Patient Profile, the ``name` element is labelled *Must Support* and has *Must Support* sub-elements `family` and `given`. When claiming conformance to this profile:
```
There is an additional '`' in ``name` that messed up rendering of this whole line.

dt-r replied:
fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -315,28 +145,39 @@ The full set of sub-elements is visible in the "Key Elements Table" or "Snapshot
+
+For example, in the AU Core Patient Profile, the ``name` element is labelled *Must Support* and has *Must Support* sub-elements `family` and `given`. When claiming conformance to this profile:
+- AU Core Responders **SHALL** correctly populate a value in `Patient.name.family` and `Patient.name.given` if the value for those sub-elements is known.
+- AU Core Requesters **SHALL** accept a Patient resource without error if `Patient.name` present and contains valid values in `family` and `given` sub-element.
```
Missing word - AU Core Requesters **SHALL** accept a Patient resource without error if `Patient.name` **_+is+_** present and contains valid values in `family` and `given` sub-element.

dt-r replied:
fixed. Also changed to Practitioner instead of Patient since Practitioner only supports those two sub-elements whereas Patient supports more.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -315,28 +145,39 @@ The full set of sub-elements is visible in the "Key Elements Table" or "Snapshot
-Some elements labelled as *Must Support* reference multiple resource types or profiles (e.g., Observation.performer). AU Core Responders **SHALL** support at least one referenced resource or profile data type for each element listed in the table below. AU Core Requesters **SHALL** support all referenced resources or profiles listed in the table below.
+Some elements labelled as *Must Support* reference multiple resource types or profiles such as `Observation.performer`. In such cases: 
+- AU Core Responders **SHALL** correctly populate the element with at least one referenced resource or allowed profile types if the value is known. 
+- AU Core Requesters **SHALL** accept resources without error if the element is present and containing any valid referenced resource or profiles.
```
I find this very confusing - it just seems overly complicated for no reason. The first sentence already qualifies that some MS elements reference multiple resource types or profiles. 'valid reference' instead of 'valid referenced resource or profiles' is much clearer and does not take the attention away from the req for an AU Core Responder to correctly populate the element with at least one valid reference .. and AU Core Requester to accept the resource if the element is present and containing any valid reference.

dt-r replied:
The pattern of leading sentence and bullet points follow the others and while I agree this is not very clear. The typo is "profile types" to be "profiles".

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -411,7 +264,94 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+         }
+    ~~~
+      
+2. For *non-coded* data elements where the applicable AU Core profile mandates a sub-element, e.g. in AU Core PractitionerRole, the sub-element `Practitioner.name.family` is mandatory, then the resource must contain the sub-element otherwise the resource will not be conformant.
```
AU Core PractitionerRole should be AU Core Practitioner.

dt-r replied:
fixed

dbojicic commented:
> > Home page - How to read this guide should be updated to include two new pages: Declaring Conformance and Actor Definitions.
> 
> fixed.

Confirmed fix.

dbojicic commented:
> 1. In 2.1.2.2 AU Core Capability Statements, change "a AU Core Requester actor" and 'a AU Core Responder actor" to use correct article "an".
> 2. Home page - Capability Statements description should be updated to use Requester and Responder rather than Client and Server.

Confirmed.

dbojicic commented:
> > Obligations on the Must Support elements inherited from core FHIR (applies to all vital signs profiles) are missing. Please add.
> 
> Added, noting that this was not added to Observation.value[x] as it is irrelevant in the profiles. It was added to Observation.valueQuantity or in the case of Blood Pressure it was not added since that profile prohibits the element Observation.value

Confirmed.

dbojicic commented:
A few minor changes needed.

dbojicic reviewed (changes requested):
Just a few tweaks needed.

dbojicic commented:
Must Support - Choice of Data Types table is incomplete: 

- Diagnostic Result Observation: Observation.value[x] & Observation.component.value[x] is missing dateTime
- AU Core Pathology Result Observation is missing
- other Observations are missing

dt-r commented:
> Must Support - Choice of Data Types table is incomplete:
> 
> * Diagnostic Result Observation: Observation.value[x] & Observation.component.value[x] is missing dateTime
> * All other Observation profiles need to be added as they have data type choices

Done.

dbojicic reviewed (commented):
Just noting here that two QA errors related to ActorDefinition.type value of 'system' are known errors that seem to be caused byR4 and R5 value sets having different values. 
Based on https://chat.fhir.org/#narrow/stream/179252-IG-creation/topic/QA.20error.20with.20ActorDefinition.20in.20R4.20IG it is not likely to be fixed. 
HL7 Europe Laboratory Report has the same issue: https://jira.hl7.org/browse/FHIR-43058
& published with errors: https://hl7.eu/fhir/laboratory/qa.html#C__zFHIR-publication_hl7-eu_temp_hl7.fhir.eu.laboratory_0.1.0-milestone_fsh-generated_resources_ActorDefinition-actor-consumer-eu-lab

dbojicic reviewed (changes requested):
Just a small change

dt-r commented:
> Just noting here that two QA errors related to ActorDefinition.type value of 'system' are known errors that seem to be caused byR4 and R5 value sets having different values. Based on https://chat.fhir.org/#narrow/stream/179252-IG-creation/topic/QA.20error.20with.20ActorDefinition.20in.20R4.20IG it is not likely to be fixed. HL7 Europe Laboratory Report has the same issue: https://jira.hl7.org/browse/FHIR-43058 & published with errors: https://hl7.eu/fhir/laboratory/qa.html#C__zFHIR-publication_hl7-eu_temp_hl7.fhir.eu.laboratory_0.1.0-milestone_fsh-generated_resources_ActorDefinition-actor-consumer-eu-lab

This is noted and is over to you, like the QA issue on CapabilityStatements to determine the appropriate means forwards as the FHIR Standards Developer.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -1,4 +1,4 @@
-### Conforming to AU Core
+### Conformin                                                                                                                                     g to AU Core
```
Spaces introduced here, please fix.

dt-r replied:
fixed

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -200,22 +200,23 @@ AU Core Profile |Must Support Element|Data Types
+AU Core Body Weight|Observation.component.value[x]|Quantity, CodeableConcept, string, boolean, integer, Range, Ratio, SampledData, time, dateTime, Period
+AU Core Heart Rate|Observation.component.value[x]|Quantity, CodeableConcept, string, boolean, integer, Range, Ratio, SampledData, time, dateTime, Period
+AU Core Respiration Rate|Observation.component.value[x]|Quantity, CodeableConcept, string, boolean, integer, Range, Ratio, SampledData, time, dateTime, Period
 AU Core Waist Circumference|Observation.component.value[x]|Quantity, CodeableConcept, string, boolean, integer, Range, Ratio, SampledData, time, Period
```
Missing change to add dateTime.

dt-r replied:
Fixed.

dbojicic commented on input/pagecontent/general-requirements.md:
```diff
@@ -200,22 +200,23 @@ AU Core Profile |Must Support Element|Data Types
 AU Core Immunization|Immunization.occurrence[x]|dateTime, string
 AU Core MedicationRequest|MedicationRequest.medication[x]|CodeableConcept, Reference
-AU Core Procedure|Procedure.performed[x]|dateTime, Period, string, Age, Range
+AU Core Procedure|Procedure.performed[x]|datetime, dateTime, Period, string, Age, Range
```
dateTime listed twice.

dt-r replied:
Fixed.

dbojicic commented on input/intro-notes/StructureDefinition-au-core-procedure-intro.md:
```diff
@@ -23,6 +23,6 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 - In an exchange with the My Health Record system `Procedure.status` is "completed"
 - The Procedure resource can represent a reason as a code with `Procedure.reasonCode`, or a reference with `Procedure.reasonReference` to a Condition or other resource.
-  - Although both are marked as must support, responders are not required to support both a code and a reference, but they **SHALL** support *at least one* of these elements
+  - Although both are marked as *Must Support* , responders are not required to support both a code and a reference, but they **SHALL** support *at least one* of these elements
```
Remove space after *Must Support*

dt-r replied:
fixed

dbojicic commented on input/intro-notes/StructureDefinition-au-core-medicationrequest-intro.md:
```diff
@@ -21,9 +21,9 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
   - When referencing a Medication resource, it is preferred the resource is [contained](http://hl7.org/fhir/R4/references.html#contained) but it may be an external resource
 - The MedicationRequest resource can represent a reason as a code with `MedicationRequest.reasonCode`, or a reference with `MedicationRequest.reasonReference` to a Condition or other resource.
-  - Although both are marked as must support, responders are not required to support both a code and a reference, but they **SHALL** support *at least one* of these elements
+  - Although both are marked as *Must Support* , responders are not required to support both a code and a reference, but they **SHALL** support *at least one* of these elements
```
Remove space after *Must Support*.

dt-r replied:
fixed in both instances

dbojicic commented:
Re Formatting of Must Support on other pages - missing changes in the following pages: 

- General Guidance > Extensibility - "additional" elements
- AU Core Encounter > Profile specific implementation guidance
- AU Core Pathology Result Observation > Profile specific implementation guidance
- AU Core Diagnostic Result Observation > Profile specific implementation guidance
- AU Core Patient > Gender Identity and related concepts
-

dbojicic reviewed (changes requested):
A few more changes.

dt-r commented:
> Re Formatting of Must Support on other pages - missing changes in the following pages:
> 
> * General Guidance > Extensibility - "additional" elements
> * AU Core Encounter > Profile specific implementation guidance
> * AU Core Pathology Result Observation > Profile specific implementation guidance
> * AU Core Diagnostic Result Observation > Profile specific implementation guidance
> * AU Core Patient > Gender Identity and related concepts

1. fixed
2. fixed
3. & 4. fixed by editing the include file on observation diagnostics guidance
5. fixed.
